tests/jballerina-unit-test/src/test/resources/test-src/query/query-action.bal

```diff
@@ -69,6 +69,39 @@ function testSimpleQueryAction2() returns int{
     return count;
 }
 
+function testSimpleQueryAction3() returns error? {
```

Comment:
Do these tests actually test the scenario in https://github.com/ballerina-platform/ballerina-lang/issues/36622? I tried running these on 2201.1.0 and it didn't seem to crash whereas the code in the issue caused a crash.

Reply:
I will check

Reply:
Updated. a54ee512c764cf67e5cd64d991a28448439772a2